src/Italy/Mews.Fiscalizations.Italy/Uniwix/Communication/UniwixClient.cs

```diff
@@ -79,52 +74,56 @@ public Task<UniwixUser> CreateUserAsync(CreateUserParameters createUserParameter
             return PostAsync<UniwixUser>(url, content);
         }
 
-        public Task<bool> VerifyCredentialsAsync()
+        public async Task<ITry<bool, ErrorResult>> VerifyCredentialsAsync()
         {
-            var url = $"{UniwixBaseUrl}/Info";
-            return ExecuteRequestAsync(url, HttpMethod.Get, content: null, responseProcessor: r => r.IsSuccessStatusCode);
+            // This endpoint returns the account information, so if the ITry is success it means the credentials are valid.
+            var result = await GetAsync<object>($"{UniwixBaseUrl}/Info");
+            return result.Map(r => true);
```

Comment:
Shouldn't this still be like this?
```suggestion
            return result.Map(r => r.IsSuccessStatusCode);
```

Reply:
This is handled in `ExecuteRequestAsync`, if anything is wrong, it will be in the Error part, otherwise we know its always true (if we got a valid response, then we know the credentials are valid).

If we ever needed this response to return the account information, all we have to do is create the model and map the response. but for now its good enough to say if we have a valid response, then the result is true (valid credentials).

Reply:
But you only changed the way exceptions are handled. So if it was possible before that it would be false here, then it's still possible.